### PR TITLE
Improve ux geo crud

### PR DESCRIPTION
**Pull request by dtrucs**

### CODE REVIEW

File: src/modules/CRUD/components/DataTable/Header/Header.js [hunk 1]
```diff
@@ -15,24 +24,44 @@ const Header = ({
...
+          ? (<Spinner size="20" />)
+          : (
+            <Tag
+              intent={Intent.PRIMARY}
+              round
+            >
+              {t('CRUD.table.count', { count: featuresList.length })}
+            </Tag>
+          )
       )}
```
Comment (mabhub) [inline src/modules/CRUD/components/DataTable/Header/Header.js]:
```jsx
      {Array.isArray(featuresList) && (
        loading
          ? <Spinner size="20" />
          : (
            <Tag intent={Intent.PRIMARY} round>
              {t('CRUD.table.count', { count: featuresList.length })}
            </Tag>
          )
      )}
```
or 
```jsx
      {Array.isArray(featuresList) && loading && <Spinner size="20" />}
      {Array.isArray(featuresList) && !!loading && (
        <Tag intent={Intent.PRIMARY} round>
          {t('CRUD.table.count', { count: featuresList.length })}
        </Tag>
      )}
```
or better:
```jsx
  // ...
  const showLoader = Array.isArray(featuresList) && loading;
  const showCount = Array.isArray(featuresList) && !loading;

  return (
    <div className="table-header">
      {/* ... */}


        {showLoader && <Spinner size="20" />}

        {showCount && (
          <Tag intent={Intent.PRIMARY} round>
            {t('CRUD.table.count', { count: featuresList.length })}
          </Tag>
        )}


      {/* ... */}
    </div>
  );
```

File: public/locales/fr/translation.json [hunk 2]
```diff
@@ -187,7 +187,8 @@
       "hideTable": "Cacher le tableau attributaire",
       "noFeature": "Aucun objet à afficher pour cette couche.",
       "readMessage": "<0>Voir</0> cet objet.",
-      "readOrUpdateMessage": "<0>Voir</0> ou <2>Modifier</2> cet objet."
+      "readOrUpdateMessage": "<0>Voir</0> ou <2>Modifier</2> cet objet.",
+      "count": "{{ count }} résultats"
```
Comment (SebCorbin) [inline public/locales/fr/translation.json]: I would name it `results :  "{{ count }} résultat"` and add a `result_plural :  "{{ count }} résultats"`

File: src/modules/CRUD/components/DataTable/Header/index.js [hunk 3]
```diff
@@ -1,19 +1,30 @@
...
+  return {
+    displayAddFeature: permissions.includes('can_add_feature'),
+  };
+};
+
+const CRUDProviderGetter = ({
+  featuresList,
+}) => ({
+  featuresList,
+});
```
Comment (mabhub) [inline src/modules/CRUD/components/DataTable/Header/index.js]:
```jsx
const CRUDProviderGetter = ({ featuresList }) => ({ featuresList });
```

but isn't this:
```jsx
const CRUDProviderGetter = ({ featuresList }) => ({ featuresList });

// ...

export default compose(
  connectCRUDProvider(CRUDProviderGetter),
)(Header); 
```
the same as:
```jsx
// const CRUDProviderGetter = ({ featuresList }) => ({ featuresList });

// ...

export default compose(
  connectCRUDProvider('featuresList'),
)(Header); 
```

File: src/modules/CRUD/components/Nav/NavItem/NavItem.js [hunk 4]
```diff
@@ -1,36 +1,51 @@
...
+      <Popover
+        content={t('CRUD.details.create')}
+        interactionKind={PopoverInteractionKind.HOVER}
+        position={Position.RIGHT}
+        boundary="window"
+      >
+        {children}
+      </Popover>
+    )
+  );
```
Comment (mabhub) [inline src/modules/CRUD/components/Nav/NavItem/NavItem.js]:
```jsx
const NavAddFeature = ({ withPopover, children, t }) => {
  if (!withPopover) {
    return children;
  }

  return (
    <Popover
      content={t('CRUD.details.create')}
      interactionKind={PopoverInteractionKind.HOVER}
      position={Position.RIGHT}
      boundary="window"
    >
      {children}
    </Popover>
  );
};

```

**Pull request merged**